src/main/scala/exercises/BalancedBrackets.scala

```diff
@@ -0,0 +1,69 @@
+package exercises
+
+import scala.annotation.tailrec
+
+class BalancedBrackets {
+
+  /*
+      Solution Logic
+      1 - separate the string that is (s) into a list of bracket pairs(stringToBracketPairs)
+          from exterior going in
+      2 - Count the number of balanced brackets in the resulting list
+      3 - if all bracket pairs in the list are balanced, then the string (s)
+          is balanced so return YES, else return NO
+     */
+
+  val listOfBalancedBraces: List[String] = List("[]", "{}", "()")
+
+  def isBalanced(s: String): String = {
+
+    @tailrec
+    def stringToBracketPairs(x: String, acc: List[String]): List[String] = {
+      val strLen: Int = x.length
+      if (strLen == 0 || strLen % 2 != 0) acc
+      else {
+        val firstAndLastChars: String = s"${x(0)}${x(strLen - 1)}"
```

Comment:
You can access the first and lay chars in a string by using .head and .last functions respectively.